src/main/kotlin/com/sourcegraph/cody/config/CodyAuthenticationManager.kt

```diff
@@ -251,7 +251,6 @@ class CodyAuthenticationManager(val project: Project) :
   override fun loadState(state: AccountState) {
     val initialAccount =
         state.activeAccountId?.let { id -> accountManager.accounts.find { it.id == id } }
-            ?: getAccounts().firstOrNull()
```

Comment:
Don't we need it anymore? 🤔

Reply:
It was actually kind of buggy behaviour, causing you to auto sign-in if you signed off.

Reply:
hah, that was an intended behaviour previously 🤣

Reply:
I know... but we had no sign-off button. Now it looks buggy if it signs you in automatically :)